blockchain/blockchain-wallet-v4-frontend
# Feat/Withdraw lock: we display amount of pending transactions based on withdraw lock

## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
How to test, make sure that you are running wallet which have ability to buy with CC, do one transaction and go to the withdraw flow, you will see pending transaction notification there
